scripts/legacy/sample_additional_info.py

```diff
@@ -174,7 +171,39 @@ def get_url_formatters_for_properties(property_mapping_path, output):
         query = (
             """SELECT * WHERE { wd:%s wdt:P1630 ?formatterUrl . }""" % prop_id
         )
-        for r in make_request(query):
+        for r in _make_request(query):
             formatters[prop_id] = r['?formatterUrl']
 
     json.dump(formatters, open(filepath, 'w'), indent=2, ensure_ascii=False)
+
+
+def query_info_for(qids_bucket, properties):
```

Comment:
This public function does not have typing annotations

Reply:
I understand, but this is legacy code. It won't be used in production, and is kept for archive purposes.